language/src/regent/optimize_index_launches.t

```diff
@@ -1349,11 +1356,11 @@ local function insert_dynamic_check(is_demand, args_need_dynamic_check, index_la
   for _, args in pairs(collected_args) do
     local stats = terralib.newlist()
 
-    local dim = call_args[args[1]].expr_type:ispace().dim
+    local dim = std.as_read(call_args[args[1]].value.expr_type).colors_symbol.symbol_type.dim
```

Comment:
I think you can replace `.colors_symbol.symbol_type` with `:colors()`.

Reply:
This is outdated since I was looking at an old commit, but the same idea applies to the new patch in a different location.

Reply:
Fixed.